SparkPlugs/main/default/aura/SparkPlugExampleForProcessRefund/SparkPlugExampleForProcessRefundController.js

```diff
@@ -0,0 +1,22 @@
+({
+	doInit : function(component, event, helper) {
+        let compEvent = $A.get('e.FDService:SparkPlugLoadedEvent');
+        compEvent.setParams({extensionPoint : component.get('v.extensionPoint')});
+        compEvent.fire();
+	},
+    processRefund2 : function(component, event, helper) {
```

Comment:
@SvetFonteva , lets get rid of names like processRefund2. Probably it should be 'handleClick'.
Lets test example as well. It should be 100% working example.